text/0000-Project-Goals-2025h1.md

```diff
@@ -0,0 +1,662 @@
+- Feature Name: N/A
+- Start Date: 2025-01-14
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: N/A
+
+## Summary
+
+Propose a slate of 40 project goals for 2025H1, including 3 flagship goals:
+
+* Continue making Rust easier to use for network systems by [**bringing the Async Rust experience closer to parity with sync Rust**](https://rust-lang.github.io/rust-project-goals/2025h1/./async.html). In 2025H1 we plan to:
+    * tell a complete story for the use of async fn in traits, unblocking wide ecosystem adoption;
+    * improve the ergonomics of `Pin`, which is frequently used in low-level async code; and
+    * prepare to support asynchronous (and synchronous) generators in the language.
+* Continue helping Rust support low-level projects by [**stabilizing compiler options and tooling used by the Rust-for-Linux project**](https://rust-lang.github.io/rust-project-goals/2025h1/./rfl.html). In 2025H1 we plan to:
+    * implement [RFC #3716](https://github.com/rust-lang/rfcs/pull/3716) to allow stabilizing ABI-modifying compiler flags to control code generation, sanitizer integration, and so forth;
+    * taking the first step towards stabilizing [`build-std`](https://rust-lang.github.io/rust-project-goals/2025h1/https://doc.rust-lang.org/cargo/reference/unstable.html#build-std) by [creating a stable way to rebuild core with specific compiler options](./build-std.html);
```

Comment:
links are still broken here...the latest commit didn't fix them all but you marked the threads as resolved